astropy/samp/hub_proxy.py

```diff
@@ -88,13 +88,12 @@ def disconnect(self):
         """
         Disconnect from the current SAMP Hub.
         """
-        self.proxy = None
+        if self.proxy is not None:
+            self.proxy.shutdown()
+            self.proxy = None
         self._connected = False
         self.lockfile = {}
 
-    def server_close(self):
```

Comment:
Can we just delete the method like this and backport to LTS? This is effectively an API change. Will this break downstream? Does anyone actually use this method?

Reply:
No, it's not documented and does not actually do anything useful.  Its mere existence is a bug :)

Reply:
In other words, any attempt to actually call this method will crash.

Reply:
Ah, okay. Then I guess we can still backport. Thanks for the clarification!